mmv1/products/cloudrun/terraform.yaml

```diff
@@ -151,6 +151,32 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           trigger_pubsub_tf: "trigger-pubsub-tf"
           trigger_auditlog_tf: "trigger-auditlog-tf"
         min_version: beta
+      - !ruby/object:Provider::Terraform::Examples
+        name: "cloud_run_service_multiple_regions"
+        primary_resource_type: "google_cloud_run_service"
+        primary_resource_id: "run_default"
+        vars:
+          myservice_service_ip: "myservice-service-ip"
+          myservice_backend: "myservice-backend"
+          myservice_lb_urlmap: "myservice-lb-urlmap"
+          myservice_ssl_cert: "myservice-ssl-cert"
+          myservice_https_proxy: "myservice-https-proxy"
+          myservice_lb_forwarding_rule: "myservice-lb-forwarding-rule"
+          myservice_neg: "myservice-neg"
+          myservice_run_app_REGIONMAN: "myservice-run-app-REGIONMAN"
+          myservice_https_urlmap: "myservice-https-urlmap"
+          myservice_http_proxy: "myservice-http-proxy"
+          myservice_https_forwarding_rule: "myservice-https-forwarding-rule"
+          project_id: "project-id"
+          domain_name: "example.com"
+          run_container_image: "us-docker.pkg.dev/cloudrun/container/hello"
+          run_regions: ["us-central1", "europe-west1"]
```

Comment:
Ah! Yes, sorry, I didn't catch this before - anything in the `vars` block here will have a random-suffix appended to it. I think for the most part, if you don't want the random suffix, you can just hardcode it to the example.

Reply:
Is there a solution where we can make this _some_ sort of variable? (Either in the Terraform layer, or the Ruby layer.)

(I feel like hard-coding this will make it harder for users to change its value, to specify the regions _they_ care about.)

Reply:
I see. That's not really what the `vars` block is intended for. The `vars` block is intended more for our testing purposes, so typically we'd put names in here that need to be unique so that as we run multiple tests in parallel they won't overlap (which is why that random suffix is appended to the end). When creating documentation, we're just going to copy the values straight into the example and be hard-coded anyway. If you wanted to create a local variable inside the example, you could do that?

Reply:
Thanks for the suggestion! I went ahead and used local variables here. 👍